plugin/phpactor.vim

```diff
@@ -33,6 +33,10 @@ if !exists('g:phpactorCompletionIgnoreCase')
     let g:phpactorCompletionIgnoreCase = 1
 endif
 
+if !exists('g:PhpactorQuickfixStrategy')
+    let g:PhpactorQuickfixStrategy = v:null
```

Comment:
```suggestion
    let g:PhpactorQuickfixStrategy = function('phpactor#quickfix#fzf')
```

We don't need `v:null` since we already handle a default.
We cound even make `s:auto_detect_strategy()` public, something like `phpactor#quickfix#default_strategy()`
So we can use it to initialize the config variable and keep the same logic.